### PR TITLE
Add instructions to extract vmlinux from /boot path

### PR DESCRIPTION
**Pull request by bgogul**

### CODE REVIEW

**Review by tiziano88 — APPROVED** [review]
Thanks Gogul! Though the formatting seems off, you will probably get a failure in presubmit. You can run `./scripts/xtask format` from within the Docker container to fix this (and then re-push).

**Comment by bgogul** [comment]
> Thanks Gogul! Though the formatting seems off, you will probably get a failure in presubmit. You can run `./scripts/xtask format` from within the Docker container to fix this (and then re-push).

Thank you, Tiziano. I fixed the formatting errors, but `cargo deny` test is failing, and I have no clue why.

**Comment by tiziano88** [comment]
Ah, it looks like that's not your fault -- just created https://github.com/project-oak/oak/issues/3832 to track. You could try to fix it yourself if you wish (feel free to send a PR), or just wait until someone from Oak picks it up, probably tomorrow :)

File: docs/development.md [hunk 1]
```diff
@@ -398,6 +398,19 @@ Response: "{\"temperature_degrees_celsius\":29}"
 [07:56:11 ✓:1,✗:0,⠇:0] └[oak-functions examples]─▶[OK] 17s
 ```
 
+## Extracing vmlinux from your Linux installation
+
+On Linux installations, you can extract the uncompressed Linux kernel ELF binary
+from the compressed kernel at `/boot/vmlinuz-$(uname -r)`. You will need the
+[extract-vmlinux](https://elixir.bootlin.com/linux/v6.2.8/source/scripts/extract-vmlinux)
```
Comment (conradgrobler) [inline docs/development.md:405]: optional: Perhaps use the latest script from the main linux repo (https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/plain/scripts/extract-vmlinux) or the github mirror (https://raw.githubusercontent.com/torvalds/linux/master/scripts/extract-vmlinux), although it is probably the same and seems unlikely to change.

**Review by conradgrobler — APPROVED** [review]
Thanks.

File: docs/development.md (hunk 1 above)
Comment (bgogul) [reply docs/development.md]: Done.

**Pull request merged**